<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnPrincipal</class>
 <widget class="QMainWindow" name="vtnPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Gestión de Personas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblNombre">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblApellido">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Apellido:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCedula">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cédula:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCedula">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellido">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNuevo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>75</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nuevo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>75</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSexo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sexo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmail">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmail">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cbSexo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hombre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mujer</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtBuscarCedula">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblCedulaBuscar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Buscar por Cedula:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>92</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBorrarRegistro">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>75</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SystemSearch"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtNombre</tabstop>
  <tabstop>txtApellido</tabstop>
  <tabstop>txtCedula</tabstop>
  <tabstop>cbSexo</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>btnNuevo</tabstop>
  <tabstop>btnLimpiar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
